### PR TITLE
Update packer templates

### PR DESCRIPTION
**Pull request by melkir**
I've used the following commands to update `packer` templates to the version `1.7.2`:
```
packer fix packer/templates/windows_2008_r2.json
packer fix packer/templates/ubuntu_1404.json
```

### CODE REVIEW

File: packer/templates/ubuntu_1404.json [hunk 1]
```diff
@@ -182,20 +192,15 @@
...
-    "iso_url": "http://old-releases.ubuntu.com/releases/14.04.0/ubuntu-14.04-server-amd64.iso",
-    "iso_checksum_type": "sha256",
+    "box_version": "0.1.12",
     "iso_checksum": "ababb88a492e08759fddcf4f05e5ccc58ec9d47fa37550d63931d0a5fa4f7388",
-    "box_version": "0.1.12"
+    "iso_checksum_type": "sha256",
+    "iso_url": "http://old-releases.ubuntu.com/releases/14.04.0/ubuntu-14.04-server-amd64.iso"
   }
-}
+}
```
Comment (melkir) [inline packer/templates/ubuntu_1404.json:206]:
```suggestion
}

```

**Review by jmartin-r7 — COMMENTED** [review]
Can you reduce this PR to only change lines required to support packer 1.7.x.  Or separate the `nodejs` change as another PR? 

`packer fix` is reordering the json for values that do not have changes. I believe the only lines required to update are for keys `ssh_wait_timeout` to become `ssh_timeout`.

The ISO url changes will need to be removed to merge here as IA64 is for `Itanium-Based Systems` and is not compatible with this project.

**Comment by jmartin-r7** [comment]
Testing shows this will also requires removal of `iso_checksum_type` keys for 1.7.x compatibility. 
```
Error: Failed to prepare build: "virtualbox-iso"

1 error occurred:
	* Deprecated configuration key: 'iso_checksum_type'. ...
```

It looks like 1.7.2 will also set newer requirements on virtualbox versions. When tested using Virtualbox `5.2.44` packer now attempts to used an option not available in that version.
```
VBoxManage: error: Unknown option: --nested-hw-virt
```
This option looks to have been [introduced in Virtualbox 6.x](https://docs.oracle.com/en/virtualization/virtualbox/6.0/admin/nested-virt.html)

The build scripts should likely be updated to check for new minimum packer version & when virtualbox is detected also validate the version is compatible.

Further testing is in progress.

**Comment by jmartin-r7** [comment]
Commit from this branch is included in #562 with credit to @melkir as provider of initial update.

**Pull request merged**